build/build-config.yml

```diff
@@ -48,8 +48,8 @@ config:
       - work-dir: "echallan-calculator"
         image-name: "echallan-calculator"
         dockerfile: "build/maven/Dockerfile"
-  - name: "builds/mGramSeva/frontend/egov-web-ui"
+  - name: "builds/mGramSeva/frontend/frontend-mgramseva"
     build:
       - work-dir: "frontend/mgramseva"
-        image-name: "egov-web-ui"
+        image-name: "frontend-mgramseva"
         dockerfile: "frontend/mgramseva/Dockerfile"
```

Comment:
Set correct dockerfile path
frontend/mgramseva/docker/Dockerfile

Reply:
Updated the Code.